contrib/gitian-build.sh

```diff
@@ -0,0 +1,199 @@
+#!/bin/bash
+
+# This file was copied from the https://github.com/cosmos/gaia project with minor changes.
+
+# symbol prefixes:
+# g_ -> global
+# l_ - local variable
+# f_ -> function
```

Comment:
I have no idea where they got it from. This is not vimscript. There are proper ways of doing this in shell. Using made up prefixes is questionable.

Reply:
👍 got your point but let's keep the diff to a minimum